fixtures/no-issues/app/controllers/application.js

```diff
@@ -1,7 +1,9 @@
 import Controller from '@ember/controller';
+import { computed } from '@ember/object';
 
-export default Controller.extend({
-  foo: computed('intl.locale', function() {
+export default class ApplicationController extends Controller {
+  @computed('intl.locale')
+  foo() {
```

Comment:
I think that's a method, not a class property 🤔

Reply:
you would have to use something like `@tracked foo;`

Reply:
nailed it! tests now fail without `classProperties` and pass with it 🙃 

PR updated